pkgs/applications/misc/yambar-hyprland-wses/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "yambar-hyprland-wses";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jonhoo";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-furHj1AAFgNiNHP9RBsVrIvrDckSKU8FXilzH9TQ99c=";
+  };
+
+  cargoHash = "sha256-/ewEgrBxRw5Xii5PX1GLKzBrZjgnzYc/Hz+M1pJpncQ=";
+
+  meta = {
+    description = "Enable Yambar to show Hyprland workspaces.";
```

```suggestion
    description = "Enable Yambar to show Hyprland workspaces";
```
Please remove the trailing period.
https://nixos.org/manual/nixpkgs/stable/#var-meta-description